packages/js/product-editor/src/components/block-slot-fill/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './block-fill';
 export * from './block-slot';
+export * from './section-actions/index';
```

Comment:
Not mandatory, but the `/index` is not really necessary.

Reply:
going to remove it. For some reason, it wasn't working without the file name. Going to remove it in a follow up

Reply:
better late than never: https://github.com/woocommerce/woocommerce/pull/46515/commits/593422e9d7c5d7cb9f295ebd87f491b28c612f11